internal/vds/vds.go

```diff
@@ -420,6 +421,14 @@ func (v VDSHandle) GetFenceMetadata(coordinates [][]float32) ([]byte, error) {
 	return buf, nil
 }
 
+func min(a, b int) int {
```

Comment:
(pure go complaint, you are good)

all I have to say: :woman_facepalming: 

(should we hide it as a local function maybe?)

Reply:
I know right! I cannot believe I had to write that method. It's already local to this file.